src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -408,20 +418,53 @@ private void setDeleteLogFileAfterCloudUpload(String deleteLogFileAfterCloudUplo
     }
 
     private void loadStateFromConfiguration(String componentName) {
-        Topics currentProcessingComponentTopics = getRuntimeConfig()
+        ProcessingFileLRU lru =  new ProcessingFileLRU(DEFAULT_MAX_FILES_TO_TRACK_PER_COMPONENT);
+
+        // Versions 2.3.0 and below
```

Comment:
nit: put these into separate methods and try to get the V2 config first. Only fall back to reading V1 config if that doesn't exist

Reply:
Done it is on a separate method and processed after v2 methods